fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabAutoCloseBanner.kt

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.compose.button.TextButton
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Top-level UI for displaying the auto close CFR banner.
+ *
+ * @param onViewOptionsClick Invoked when the user clicks to view the auto close options.
+ * @param onDismiss Invoked when the user clicks to dismiss the banner.
+ */
+@Composable
+fun TabAutoCloseBanner(
+    onViewOptionsClick: () -> Unit,
+    onDismiss: () -> Unit,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .background(FirefoxTheme.colors.layer1)
+            .padding(start = 11.dp, top = 21.dp, end = 16.dp),
+    ) {
+        Text(
+            text = stringResource(id = R.string.tab_tray_close_tabs_banner_message),
```

Comment:
I'm confirming the real text style with Design. Won't land this until I hear back.